tests/topotests/bfd-vrf-topo1/test_bfd_vrf_topo1.py

```diff
@@ -46,6 +46,9 @@
 # Required to instantiate the topology builder class.
 from mininet.topo import Topo
 
+pytestmark = [
+    pytest.mark.bfdd
```

Comment:
doesn't this one use bgp too?

Reply:
I plan to come around for a bgp fixup all in one pass.  I tried to mark the really obvious to me when I was in there for bfdd in this case